Code Samples/Fib/.vscode/tasks.json

```diff
@@ -1,50 +1,30 @@
 {
-    // See https://go.microsoft.com/fwlink/?LinkId=733558
+        // See https://go.microsoft.com/fwlink/?LinkId=733558
     // for the documentation about the tasks.json format
     "version": "2.0.0",
     "tasks": [
         {
             "label": "build",
             "type": "shell",
+            "command": "make",
             "group": {
                 "kind": "build",
                 "isDefault": true
             },
-            "presentation": {
-                "echo": true,
-                "reveal": "always",
-                "focus": false,
-                "panel": "shared"
+            "problemMatcher": ["$gcc"],
+            "options": {
+                "cwd": "${workspaceFolder}/Code Samples/Fib"
```

Comment:
What workspace is the user of this sample supposed to open? I think they're supposed to open the "Fib" workspace folder, because otherwise I'm not sure how the /.vscode/tasks.json would be picked up.